### DISCUSSION TITLE
Trouble Installing Security Onion ISO to Dell PowerEdge R740xd

### DISCUSSION THREAD

**Discussion by Pelias32** [Security-Onion-Solutions/securityonion#8492] [General]
I have had problems for the past 2 months, I have tried everything I can think of in order to get Security Onion installed on this brand new Server we purchased. This is critical due to the old one's power supply just failed. I have attached the screenshot I am getting. 

I have tried setting up the RAID differently, UEFI/BIOS boot options. I have flashed multiple flash drives with either Rufus or BalenaEtcher like suggested and I still keep getting this error. Any help would be greatly appreciated I have searched through the forums and found others have had similar issues in the past, but appears they are to be "fixed" in the latest updates, but I have tried the last 3 updates with no luck.

![20220808_105922](https://user-images.githubusercontent.com/110841417/183458065-beccc573-f687-4c9a-8c71-3687fead4fd3.jpg)

**Comment by isaacgolding** [comment]
Out of curiosity have you tried any other install programs,  EG a windows 10 DVD/USB or another operating system?  I don't have any good answers based on what you are showing...but I would personally try another installer to see if the other system has problems as well.

**Reply by Pelias32** [reply]
I have tried other installers, I am able to install Windows Server and Linux without any issues, but fails every time I attempt to install Security Onion.

**Reply by Pelias32** [reply]
Just to clarify I installed "Cent OS 7 basic" without any issues as well as 2019 Windows Server

**Reply by dougburks** [reply]
> Just to clarify I installed "Cent OS 7 basic" without any issues

If that's the case, then you can just install Security Onion on top of a normal CentOS 7 installation:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

**Reply by Pelias32** [reply]
> > Just to clarify I installed "Cent OS 7 basic" without any issues
> 
> If that's the case, then you can just install Security Onion on top of a normal CentOS 7 installation: https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos


I am being instructed that they would like to install it as an all in one and they do not understand why it won't install.

**Reply by TOoSmOotH** [reply]
@Pelias32 The ISO is simply a CentOS 7 minimal install. Take the defaults on swap and efi etc. Then make a 300GB / and put the rest in /nsm. Set up the network and do a network install. Not sure why it is failing but the reason we left the network option open was for these types of cases specifically.

**Reply by Pelias32** [reply]
> @Pelias32 The ISO is simply a CentOS 7 minimal install. Take the defaults on swap and efi etc. Then make a 300GB / and put the rest in /nsm. Set up the network and do a network install. Not sure why it is failing but the reason we left the network option open was for these types of cases specifically.

Thank you for the reply, I will give this an attempt. I understand most of what you instructed to do.

**Comment by xfaith** [comment]
Here is a full conversation about the issue: https://github.com/Security-Onion-Solutions/securityonion/issues/1825

Looks like it still happening to folks.  I know my dells R740xd install, Ill see if my engineers had any issues.  
Not sure if it would help, but I would review all Drivers/Firmware for the system and make sure they are up to date (even new systems sometime are not updated)

**Reply by Pelias32** [reply]
Thank you, would love to hear what your engineers come back with to see if they had any issues.

**Reply by reyesj2** [reply]
Hi, can I ask why you're using a bootable usb and not just mounting the ISO directly from the iDRAC virtual console?

**Reply by reyesj2** [reply]
I can try to dig it up tomorrow, but I think there was an issue with the file sizes when using certain programs to create a bootable iso.

**Reply by xfaith** [reply]
Thanks for chiming in @reyesj2

**Reply by Pelias32** [reply]
> Hi, can I ask why you're using a bootable usb and not just mounting the ISO directly from the iDRAC virtual console?

I have tried to do it through the iDRAC as well still fails

**Reply by Pelias32** [reply]
> Thanks for chiming in @reyesj2

Any update?

**Comment by Pelias32** [comment]
I have tried multiple different RAIDS as well, anyone have any insight on what they used and what worked for them

**Comment by Pelias32** [comment]
![20220808_105922](https://user-images.githubusercontent.com/110841417/183724985-632c91f0-6883-40ef-a831-8e052aa223a6.jpg)

I am unsure if I am allowed to post links, but this is a video of the loading screen and the screen until the install fails - https://www.filemail.com/d/dhulzctmxpksliz

**Comment by reyesj2** [comment]
Is there any customization you did to this ISO or did you get it directly from Security Onion? Try redownloading and verify the hash before attempting install

**Reply by Pelias32** [reply]
I got the ISO directly from Security Onion. I used the balenaEtcher just like recommended and still no luck. It is very odd because if I use my hyper-v host and load it as a virtual machine on there it installs. I do not know why this brand new server with nothing loaded on it is having such issues. All drivers are also updated on the new server.
